### ISSUE TITLE
Jenkins marian-regression-tests #253 failed

### ISSUE THREAD

**Issue by tomekd** [marian-nmt/marian-dev#245]
Build 'marian-regression-tests' is failing!

Last 50 lines of build output:

```
[...truncated 43.78 KB...]
[06/05/2018 16:34:46] Checking directory: tests/training/pretrain
[06/05/2018 16:34:46] Running setup script
[06/05/2018 16:34:46] Running tests/training/pretrain/test_weights_from_pretrained_model.sh ... OK
[06/05/2018 16:35:21] Test took 00:00:35.374s
[06/05/2018 16:35:21] Checking directory: tests/training/pretrain/model
[06/05/2018 16:35:21] Checking directory: tests/training/basics
[06/05/2018 16:35:21] Running setup script
[06/05/2018 16:35:21] Running tests/training/basics/test_mini_batch_fit.sh ... OK
[06/05/2018 16:36:34] Test took 00:01:12.524s
[06/05/2018 16:36:34] Running tests/training/basics/test_sqlite_random_seed.sh ... OK
[06/05/2018 16:37:24] Test took 00:00:49.924s
[06/05/2018 16:37:24] Running tests/training/basics/test_sqlite.sh ... OK
[06/05/2018 16:38:08] Test took 00:00:44.017s
[06/05/2018 16:38:08] Running tests/training/basics/test_toy_vocab.sh ... OK
[06/05/2018 16:38:49] Test took 00:00:41.528s
[06/05/2018 16:38:49] Running tests/training/basics/test_translation_script.sh ... OK
[06/05/2018 16:39:20] Test took 00:00:30.997s
[06/05/2018 16:39:20] Running tests/training/basics/test_valid_script.sh ... OK
[06/05/2018 16:40:08] Test took 00:00:47.729s
[06/05/2018 16:40:08] Checking directory: tests/training/basics/sqlite
[06/05/2018 16:40:08] Checking directory: tests/training/basics/valid
[06/05/2018 16:40:08] Checking directory: tests/training/basics/toy
[06/05/2018 16:40:08] Checking directory: tests/training/basics/batch_fit
[06/05/2018 16:40:08] Checking directory: tests/training/basics/trans
[06/05/2018 16:40:08] Checking directory: tests/training/basics/sqlite_seed
[06/05/2018 16:40:08] Checking directory: tests/examples
[06/05/2018 16:40:08] Checking directory: tests/examples/iris
[06/05/2018 16:40:08] Running setup script
[06/05/2018 16:40:08] Running tests/examples/iris/test_iris.sh ... OK
[06/05/2018 16:40:09] Test took 00:00:0.897s
[06/05/2018 16:40:09] Checking directory: tests/examples/mnist
[06/05/2018 16:40:09] Running setup script
[06/05/2018 16:40:09] Running tests/examples/mnist/test_mnist_ffnn.sh ... failed
[06/05/2018 16:40:48] Test took 00:00:38.850s
[06/05/2018 16:40:48] Running tests/examples/mnist/test_mnist_lenet.sh ... skipped
[06/05/2018 16:40:48] Test took 00:00:0.025s
---------------------
Skipped:
  - tests/models/char-s2s/test_char_s2s.sh
  - tests/models/char-s2s/test_compare_scores.sh
  - tests/training/restart-optimizer/test_adam_params_async.sh
  - tests/training/restart-optimizer/test_adam_params_sync.sh
  - tests/training/multi-gpu/test_async_sgd.sh
  - tests/examples/mnist/test_mnist_lenet.sh
Failed:
  - tests/examples/mnist/test_mnist_ffnn.sh
---------------------
Ran 67 tests in 00:27:22.620s, 60 passed, 6 skipped, 1 failed
Build step 'Execute shell' marked build as failure

```

Changes since last successful build:
- [Marcin Junczys-Dowmunt] 54d44c8dde64f08cb02f67fd6feece9033d8f9c3 - adjust tests/training/restart/test_restarting_stalled_validators.sh

- [Marcin Junczys-Dowmunt] 025feb5b778e20a430cd95a4fed99fa4f21b119c - adjust

- [Marcin Junczys-Dowmunt] d41ebec521e787f89d24efdfd0d851611a35b293 - adjust tests/training/restart/test_restoring_validation.sh

- [Marcin Junczys-Dowmunt] 97816d8597ecb0372a50677bd66a125daea2c488 - adjust tests/training/weights/test_validation.sh

- [Marcin Junczys-Dowmunt] 7509d7b7ab6eb02aa4a466a0e3cc6c3311a311d9 - adjust tests for final evaluation

- [Marcin Junczys-Dowmunt] 7b28da31c97635f50da842409171d4a8bb2a308e - adjust validation set for epochs

- [Marcin Junczys-Dowmunt] c86ccb046902512a6704f7359b25de3fc821287a - adjust validation format for tests

[View full output](http://vali.inf.ed.ac.uk/jenkins/job/marian-regression-tests/253/)

**Comment by tomekd** [comment]
Build was fixed!
